apps/back-office/src/app/(dashboard)/settings/components/product/ProductSection.tsx

```
@@ -0,0 +1,261 @@
+          <Input
+            className="w-full rounded-[6px] border h-[42px] mt-[12px]"
+            value={field.state.value}
+            onChange={(e) => field.handleChange(e.target.value)}
+            onBlur={() => form.handleSubmit()}
+          />
+        )}
+      />
+      <div className="flex items-center gap-2">
+        <form.Field
+          name="price"
+          children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)